<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1222</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cистема абонентов мобильного оператора </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListWidget" name="listWidget_user"/>
    </item>
    <item>
     <widget class="QCustomPlot" name="widget_plot" native="true"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_add">
        <property name="text">
         <string>add</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/icon/user_add.png</normaloff>:/icon/user_add.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_edit">
        <property name="text">
         <string>edit</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/icon/user_edit.png</normaloff>:/icon/user_edit.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_delete">
        <property name="text">
         <string>delete</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/icon/user_delete.png</normaloff>:/icon/user_delete.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_info">
        <property name="text">
         <string>info</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/icon/user_info.png</normaloff>
          <normalon>:/icon/icon/user_info.png</normalon>:/icon/user_info.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_plot">
        <property name="text">
         <string>plot</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/icon/plot.png</normaloff>:/icon/plot.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_sort">
        <property name="text">
         <string>sort</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/icon/sort.png</normaloff>
          <normalon>:/icon/icon/sort.png</normalon>:/icon/sort.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_open_file">
        <property name="text">
         <string>open</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/icon/open_file.png</normaloff>
          <normalon>:/icon/icon/open_file.png</normalon>:/icon/open_file.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>QCustomPlot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
